### PR TITLE
Fixes #7621: cf-promises can hang for about 30 seconds looking up default server name \"rudder\"

### PR DESCRIPTION
**Pull request by jooooooon**
https://www.rudder-project.org/redmine/issues/7621

Replacing previous PR: https://github.com/Normation/rudder-techniques/pull/819

### CODE REVIEW

**Comment by jooooooon** [comment]
@peckpeck Please review

File: techniques/system/common/1.0/cf-served.st [hunk 1]
```diff
@@ -98,15 +98,12 @@ bundle common def
         string  => translatepath("${sys.workdir}/policy_server.dat"),
         comment => "Path to file containing address to policy server";
 
-   # Use localhost as default policy_server if no policy_server.dat exists
-    policy_server_set::
+      # Use "rudder" as default policy_server if no policy_server.dat exists
       "policy_server"
-        string  => readfile("${policy_server_file}", 2048),
+        string  => ifelse(fileexists("${def.policy_server_file}"),
+                          readfile("${policy_server_file}", 2048),
```
Comment (peckpeck) [inline techniques/system/common/1.0/cf-served.st]: Why isn't the variable named the same way?

Comment (jooooooon) [inline techniques/system/common/1.0/cf-served.st]: No good reason. Both work since it's a local variable, but that is confusing. I'll fix it.

**Comment by jooooooon** [comment]
PR updated

**Comment by peckpeck** [comment]
Good, merging

**Pull request merged**